.pipelines/vhd-build-publish-windows.yaml

```diff
@@ -0,0 +1,38 @@
+trigger: none
+pr: none
+
+stages:
+
+  - stage: build_2019_containerd
+    dependsOn: []
```

Comment:
Why separate parallel Stages vs [job matrix](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration)?

Reply:
A few reasons 
- I found a lot more examples of multi-stage pipelines vs using job matrix
- So that we can opt to run only some stages when triggering the pipeline without needed to update the yml (like it we only needed to build/relesase a subsete of VHDs)
- I plan on adding in a manual gate / approval step which I think needs to be its own stage